File: Java Pattern Programs/Series Patterns/Pattern 7.java
class FibonacciNumbersSeries
{
 public static void main(String agrs[])
 {
  int n=15;//size or upper limit
  int a=0; 
  int b=1; 
  int c=a+b;
  
  while(c<=n)
  {
  System.out.print(" "+c);
  c=a+b;
  a=b;
  b=c;
  }
 }
}

/*
 OUTPUT :

 1 1 2 3 5 8 13
 
 **/

File: CSharpPatternPrograms/SeriesPatterns/Pattern 7.cs
using System;

class FibonacciNumbersSeries
{
    public static void Main(string[] agrs)
    {
        int n = 15; //size or upper limit
        int a = 0;
        int b = 1;
        int c = a + b;

        while (c <= n)
        {
            Console.Write(" " + c);
            c = a + b;
            a = b;
            b = c;
        }
        Console.ReadKey(true);
    }
}

/*
 OUTPUT :

 1 1 2 3 5 8 13

 **/
